include/z64cutscene.h

```diff
@@ -3,286 +3,501 @@
+    /* 0x005E */ CS_CMD_ACTOR_CUE_1_11,
+    /* 0x0069 */ CS_CMD_ACTOR_CUE_0_10 = 0x69,
+    /* 0x006A */ CS_CMD_ACTOR_CUE_2_9,
+    /* 0x006B */ CS_CMD_ACTOR_CUE_0_11,
+    /* 0x006C */ CS_CMD_ACTOR_CUE_3_10,
+    /* 0x006D */ CS_CMD_UNIMPLEMENTED_6D,
+    /* 0x006E */ CS_CMD_ACTOR_CUE_0_12,
+    /* 0x006F */ CS_CMD_ACTOR_CUE_7_3,
+    /* 0x0070 */ CS_CMD_UNIMPLEMENTED_70,
+    /* 0x0071 */ CS_CMD_UNIMPLEMENTED_71,
+    /* 0x0072 */ CS_CMD_ACTOR_CUE_7_4,
+    /* 0x0073 */ CS_CMD_ACTOR_CUE_6_5,
+    /* 0x0074 */ CS_CMD_ACTOR_CUE_1_12,
+    /* 0x0075 */ CS_CMD_ACTOR_CUE_2_10,
+    /* 0x0076 */ CS_CMD_ACTOR_CUE_1_13,
+    /* 0x0077 */ CS_CMD_ACTOR_CUE_0_13,
+    /* 0x0078 */ CS_CMD_ACTOR_CUE_1_14,
+    /* 0x0079 */ CS_CMD_ACTOR_CUE_2_11,
+    /* 0x007B */ CS_CMD_ACTOR_CUE_0_14 = 0x7B,
+    /* 0x007C */ CS_CMD_FADE_SEQ,
+    /* 0x007D */ CS_CMD_ACTOR_CUE_1_15,
+    /* 0x007E */ CS_CMD_ACTOR_CUE_2_12,
+    /* 0x007F */ CS_CMD_ACTOR_CUE_3_11,
+    /* 0x0080 */ CS_CMD_ACTOR_CUE_4_7,
+    /* 0x0081 */ CS_CMD_ACTOR_CUE_5_5,
+    /* 0x0082 */ CS_CMD_ACTOR_CUE_6_6,
+    /* 0x0083 */ CS_CMD_ACTOR_CUE_1_16,
+    /* 0x0084 */ CS_CMD_ACTOR_CUE_2_13,
+    /* 0x0085 */ CS_CMD_ACTOR_CUE_3_12,
+    /* 0x0086 */ CS_CMD_ACTOR_CUE_7_5,
+    /* 0x0087 */ CS_CMD_ACTOR_CUE_4_8,
+    /* 0x0088 */ CS_CMD_ACTOR_CUE_5_6,
+    /* 0x0089 */ CS_CMD_ACTOR_CUE_6_7,
+    /* 0x008A */ CS_CMD_ACTOR_CUE_0_15,
+    /* 0x008B */ CS_CMD_ACTOR_CUE_0_16,
+    /* 0x008C */ CS_CMD_TIME,
+    /* 0x008D */ CS_CMD_ACTOR_CUE_1_17,
+    /* 0x008E */ CS_CMD_ACTOR_CUE_7_6,
+    /* 0x008F */ CS_CMD_ACTOR_CUE_9_0,
+    /* 0x0090 */ CS_CMD_ACTOR_CUE_0_17,
+    /* 0x03E8 */ CS_CMD_DESTINATION = 0x03E8,
+    /* 0xFFFF */ CS_CMD_END = 0xFFFF
+} CutsceneCmd;
 
-typedef struct {
-    /* 0x0 */ CutsceneCameraPoint* atPoints;
-    /* 0x4 */ CutsceneCameraPoint* eyePoints;
-    /* 0x8 */ s16 relativeToPlayer;
-} CutsceneCameraMove; // size = 0xC
+#define CS_CMD_CONTINUE 0
+#define CS_CMD_STOP -1
+
+// TODO: fix ZAPD to use new names and delete these:
+#define CS_CMD_CAM_EYE CS_CMD_CAM_EYE_POINTS
+#define CS_CMD_CAM_AT CS_CMD_CAM_AT_POINTS
+#define CS_CMD_SET_LIGHTING CS_CMD_LIGHT_SETTING
+#define CS_CMD_CAM_EYE_REL_TO_PLAYER CS_CMD_CAM_EYE_POINTS_REL_TO_PLAYER
+#define CS_CMD_CAM_AT_REL_TO_PLAYER CS_CMD_CAM_AT_POINTS_REL_TO_PLAYER
+#define CS_CMD_09 CS_CMD_RUMBLE_CONTROLLER
+#define CS_CMD_SET_PLAYER_ACTION CS_CMD_PLAYER_CUE
+#define CS_CMD_SCENE_TRANS_FX CS_CMD_TRANSITION
+#define CS_CMD_TEXTBOX CS_CMD_TEXT
+#define CS_CMD_PLAYBGM CS_CMD_PLAY_SEQ
+#define CS_CMD_STOPBGM CS_CMD_STOP_SEQ
+#define CS_CMD_FADEBGM CS_CMD_FADE_SEQ
+#define CS_CMD_SETTIME CS_CMD_TIME
+#define CS_CMD_SET_ACTOR_ACTION_1 CS_CMD_ACTOR_CUE_0_0
+#define CS_CMD_SET_ACTOR_ACTION_2 CS_CMD_ACTOR_CUE_1_0
+#define CS_CMD_SET_ACTOR_ACTION_4 CS_CMD_ACTOR_CUE_2_0
+#define CS_CMD_SET_ACTOR_ACTION_3 CS_CMD_ACTOR_CUE_3_0
+#define CS_CMD_SET_ACTOR_ACTION_5 CS_CMD_ACTOR_CUE_4_0
+#define CS_CMD_SET_ACTOR_ACTION_6 CS_CMD_ACTOR_CUE_5_0
```

Comment:
Is ACTION_4 and ACTION_3 supposed to be swapped here? The rest of this is ordered 1-10 (0-9) here